py3status/modules/bluetooth.py

```diff
@@ -60,14 +60,40 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 10
-    device_separator = '|'
     format = DEFAULT_FORMAT
     format_device = '{name}'
+    format_separator = '\|'
+
+    class Meta:
+        def deprecate_function(config):
+            if (not config.get('format_separator') and
+                    config.get('device_separator')):
+                sep = config.get('device_separator')
+                sep = sep.replace('\\', '\\\\')
+                sep = sep.replace('[', '\[')
+                sep = sep.replace(']', '\]')
+                sep = sep.replace('|', '\|')
+
+                return {'format_separator': sep}
+            else:
+                return {}
+
+        deprecated = {
+            'function': [
+                {
+                    'function': deprecate_function,
+                },
+            ],
```

Comment:
seeing as this code has appeared twice (also in #905) we should just make it a deprecation function so we don't have all this ugliness in a module.